### PR TITLE
Jsp 283 scivias strategy pages

### PR DESCRIPTION
**Pull request by Dhul-Husni**
WIP

### CODE REVIEW

File: app/javascript/packs/application.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ require("@rails/ujs").start()
 require("@rails/activestorage").start()
 require("channels")
 
+require("../stylesheets/application.scss");
```
Comment (dcassanego) [inline app/javascript/packs/application.js]: 'require' is not defined.

File: app/javascript/stylesheets/tailwind.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+module.exports = {
```
Comment (dcassanego) [inline app/javascript/stylesheets/tailwind.js]: 'module' is not defined.

File: app/javascript/stylesheets/tailwind.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+  theme: {
+    extend: {},
+  },
+  variants: {},
+  plugins: [
+    require('@tailwindcss/custom-forms')
```
Comment (dcassanego) [inline app/javascript/stylesheets/tailwind.js]: 'require' is not defined.

File: app/controllers/api_controller.rb [hunk 4]
```diff
@@ -4,8 +4,9 @@ class ApiController < ApplicationController
   def api_authenticate
     # Uncomment when debugging auth issues
     # print_auth_debugging(request)
-    @current_account = ApiAccessCredential.find_by(access_id: ApiAuth.access_id(request))
-    head(:unauthorized) unless @current_account && ApiAuth.authentic?(request, @current_account.secret_key)
+    # TODO: DHUL UNCOMMENT
```
Comment (MemunaHaruna) [inline app/controllers/api_controller.rb:7]: Hey @Dhul-Husni reminder to un-comment the relevant lines here

File: app/controllers/application_controller.rb [hunk 5]
```diff
@@ -20,13 +20,14 @@ class ApplicationController < ActionController::Base
   # these methods (and therefore has a naming conflict).  This is documented
   # here:
   # https://activeadmin.info/13-authorization-adapter.html#using-the-pundit-adapter
-  after_action  :verify_authorized,
-                except: :index,
-                unless: :active_admin_controller?
+  # todo: dhul uncomment
```
Comment (MemunaHaruna) [inline app/controllers/application_controller.rb:23]: same here

File: app/controllers/api/projects_controller.rb [hunk 6]
```diff
@@ -0,0 +1,12 @@
+class Api::ProjectsController < ApiController
+  def draft_conversations
+    project = Project.find_by(project_code: params[:project_code])
+    conversations = ConversationBuilder.where(project: project)
+
+    draft_conversations = conversations.map do |draft|
+      { name: draft.conversation_name, url: conversation_builder_wizard_index_url(draft) }
+    end
+
+    render json: draft_conversations, status: :ok
```
Comment (MemunaHaruna) [inline app/controllers/api/projects_controller.rb:10]: @Dhul-Husni it's a good idea for us to start using jbuilder (https://github.com/rails/jbuilder) to return JSON responses in Scivias. I forgot to add the jbuilder gem to the Gemfile when I created the PR for ApiAuth.

**Comment by MemunaHaruna** [comment]
Looks good so far! @Dhul-Husni

**Comment by Dhul-Husni** [comment]
reopening

**Pull request closed without merging**